surveyor/collector_statz.go

```diff
@@ -155,6 +160,30 @@ func buildDescs(sc *StatzCollector) {
 	sc.descs.GatewayRecvBytes = newPromDesc("gateway_recv_bytes", "Number of messages sent over the gateway gauge", gatewayLabels)
 	sc.descs.GatewayNumInbound = newPromDesc("gateway_inbound_msg_count", "Number inbound messages through the gateway gauge", gatewayLabels)
 
+	// Surveyor
+	sc.surveyedCnt = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: prometheus.BuildFQName("nats", "survey", "surveyed_count"),
+		Help: "Number of remote hosts successfully surveyed gauge",
+	}, []string{})
+
+
+	sc.expectedCnt = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: prometheus.BuildFQName("nats", "survey", "expected_count"),
+		Help: "Number of remote hosts expected to responded gauge",
+	}, []string{})
+
+
+	sc.pollTime = prometheus.NewSummaryVec(prometheus.SummaryOpts{
+		Name:prometheus.BuildFQName("nats", "survey", "duration_seconds"),
+		Help: "Time it took to gather the surveyed data histogram",
+	}, []string{})
+
+
+	sc.pollErrCnt = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name:prometheus.BuildFQName("nats", "survey", "poll_error_count"),
+		Help:"The number of times the poller encountered errors counter",
+	}, []string{})
```

Comment:
It's annoying to have 2 ways to define these metrics but conceptually they are different kinds of thing, one represents remote data one local state so using the more typical prom method for these counters is a LOT easier to use - else we would need to maintain all the data in process and then each time instantiate new metrics.  This way prom classes does all the work - especially handy with the summary one.